### PR TITLE
Protofolio Toolbar/Headline PR

### PR DESCRIPTION
**Pull request by Nessrine88**
This pull request should contain a responsive web page presenting  the toolbar and the headline of my protofolio.
It includes:
index.html
style.css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,150 @@
...
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+# 📖 [Hello Microverse] <a name="about-project"></a>
```
Comment (Qoosim) [inline README.md]: - [ ] Kindly give a descriptive title to your project. Your project is no longer hello microverse. You're building your portfolio now and that should reflect in your README file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Protofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+    <section class="toolbar">
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to use `header` tag for this section. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            </ul>
+        </nav>
+
+    </section>
+
+    <section class="headline">
+        <h1 class="primary">
+            Hey there. I’m Nessrine.
+            <br>
+            <span>I’m a software developer</span>
```
Comment (Qoosim) [inline index.html:28]: - [ ] This is a subtitle to the main title on your page - `Hey there, I'm Nessrine`. The subtitle (if existing) uses a header tag `hx (examples: <h2>,<h3>, ... <h6> etc...)`. Kindly change your subtitle to any of these `hx` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        <p class="supporting-text">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </p>
+        <div class="social-media">
+            <img src="images/social-media/github.svg" alt="github-icon">
+            <img src="images/social-media/Linkedin icon.svg" alt="linkedin-icon">
+            <img src="images/social-media/angellist.svg" alt="angelist-icon">
+            <img src="images/social-media/twitter.svg" alt="twitter-icon">
+            <img src="images/social-media/medium.svg" alt="medium-icon">
```
Comment (Qoosim) [inline index.html]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly address that too.

File: test.md [hunk 5]
```diff
@@ -0,0 +1 @@
+this should be ignored
```
Comment (Qoosim) [inline test.md]: - [ ] Your test.md file is not expected to be part of this project again. Kindly remove it or ignore it by git as you did in your first project at Microverse.

**Review by Qoosim — CHANGES_REQUESTED** [review]

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Nessrine88, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Nessrine88, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented previous suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] You have a horizontal scrollbar on your webpage. Kindly fix the issue.

https://github.com/Nessrine88/Protofolio/assets/70483835/ea3f7159-263a-4a0b-8d44-2bde65dadf56


- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**